integration-tests/chopsticks/src/tests/xcm/limitedReserveTransfer/index.test.ts

```diff
@@ -0,0 +1,124 @@
+import { describe, beforeEach, it, afterEach } from 'vitest'
+import type { KeyringPair } from '@polkadot/keyring/types'
+import { sendTransaction, withExpect } from '@acala-network/chopsticks-testing'
+
+import { createBlock, setStorage } from '../../../network/utils.js'
+import { validateBalanceWithPrecision, hexAddress } from '../../../helper/utils.js'
+import { testPairsLimitedReserveTransfers } from './config.js'
+import { Config } from '../../../network/types.js'
+import { setupNetwork, shutDownNetwork } from '../../../network/utils.js'
+
+describe.each(testPairsLimitedReserveTransfers)(
+	'Limited Reserve Transfers',
+	{ timeout: 30_000 },
+	async ({ network, storage, accounts, query, sovereignAccount, txContext, config }) => {
+		let senderContext: Config
+		let receiverContext: Config
+		let relayContext: Config
+		let senderAccount: KeyringPair
+		let receiverAccount: KeyringPair
+		const { desc, precision } = config
+
+		beforeEach(async () => {
+			const { receiver, sender, relay } = network
+
+			const { receiverChainContext, senderChainContext, relayChainContext } = await setupNetwork(
+				relay,
+				sender,
+				receiver
+			)
+
+			relayContext = relayChainContext
+			senderContext = senderChainContext
+			receiverContext = receiverChainContext
+
+			const { receiverStorage, senderStorage, relayStorage } = storage
+			await setStorage(senderContext, senderStorage)
+			await setStorage(receiverContext, receiverStorage)
+			await setStorage(relayContext, relayStorage)
+
+			const { senderAccount: a, receiverAccount: b } = accounts
+			senderAccount = a
+			receiverAccount = b
+		}, 20_000)
+
+		afterEach(async () => {
+			try {
+				await shutDownNetwork([senderContext, receiverContext, relayContext])
+			} catch (error) {
+				if (!(error instanceof TypeError)) {
+					console.error(error)
```

Comment:
How is this different then just not `catch`ing the error at all? It will be printed to `consoler.err` anyway. Or you don't want to print type errors to console error? If not, why not?